src/components/pages/rules/flyouts/ruleEditor/ruleEditor.scss

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+@import 'src/styles/themes';
+@import 'src/styles/mixins';
+
+.new-rule-flyout-container {
+
+  .rule-property-container {
+    border: none;
+    @include rem-fallback(margin-top, 20px);
+  }
+
+  .devices-affected{
+    display: flex;
+    flex-flow: row nowrap;
+    position: relative;
+    @include rem-fallback(top, 18.5px);
+    @include rem-fallback(height, 67px);
+  }
+
+  @include themify($themes) {
+    .devices-affected-dynamic {
+      @include rem-fallback(font-size, 48px);
+      font-weight: bold;
+      color: themed('colorHighest');
```

Comment:
`.devices-affected-dynamic` should be defined above with static props while the themable props are only in the themify.